modules/lib/src/main/resources/assets/js/app/settings/dialog/ProjectSelectionDialog.ts

```diff
@@ -92,6 +92,7 @@ export class ProjectSelectionDialog
     open() {
         if (!this.updateOnOpen) {
             this.showItems();
+            this.addFocusToFirstProjectListElement();
```

Comment:
Will work only if no update needed. Try to create some layer or project in Settings app, and then open dialog again - focus will not be set

Reply:
Dialog has a lazy loading for a projects, so if no updates on existing projects/layers were made then it won't load entire list of projects, however if updated event happened then dialog will refetch it's items and focus will have to be set after that